swagger.yaml

```
@@ -3300,6 +3335,126 @@ paths:
         "500":
           $ref: "#/components/responses/internal_server_error"
 
+  /scripts:
+    get:
+      tags:
+        - Cardano » Scripts
+      summary: Scripts
+      description: List of scripts.
+      parameters:
+        - in: query
+          name: count
+          required: false
+          schema:
+            type: integer
+            maximum: 100
+            default: 100
+          description: The number of results displayed on one page.
+        - in: query
+          name: page
+          required: false
+          schema:
+            type: integer
+            default: 1
+          description: The page number for listing the results.
+        - in: query
+          name: order
+          required: false
+          schema:
+            type: string
+            enum: [asc, desc]
+            default: asc
+          description: |
+            The ordering of items from the point of view of the blockchain,
+            not the page listing itself. By default, we return oldest first, newest last.
+      responses:
+        "200":
+          description: Return list of scripts
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/scripts"
+        "400":
+          $ref: "#/components/responses/bad_request"
+        "403":
+          $ref: "#/components/responses/unauthorized_error"
+        "404":
+          $ref: "#/components/responses/not_found"
+        "429":
+          $ref: "#/components/responses/overusage_limit"
+        "418":
+          $ref: "#/components/responses/autobanned"
+        "500":
+          $ref: "#/components/responses/internal_server_error"
+
+  /scripts/{script_hash}:
+    get:
+      tags:
+        - Cardano » Scripts
+      summary: Specific script
+      description: Information about a specific script
+      parameters:
+        - in: path
+          name: script_hash
+          required: true
+          schema:
+            type: string
+          description: Hash of the script
+          example: "e1457a0c47dfb7a2f6b8fbb059bdceab163c05d34f195b87b9f2b30e"
```

```suggestion
        - in: path
          name: script_hash
          required: true
          schema:
            type: string
          description: Hash of the script
          example: "e1457a0c47dfb7a2f6b8fbb059bdceab163c05d34f195b87b9f2b30e"
        - in: query
          name: count
          required: false
          schema:
            type: integer
            maximum: 100
            default: 100
          description: The number of results displayed on one page.
        - in: query
          name: page
          required: false
          schema:
            type: integer
            default: 1
          description: The page number for listing the results.
        - in: query
          name: order
          required: false
          schema:
            type: string
            enum: [asc, desc]
            default: asc
          description: |
            The ordering of items from the point of view of the blockchain,
            not the page listing itself. By default, we return oldest first, newest last.
```

Pagination is missing.